petab/migrations.py

```diff
@@ -0,0 +1,116 @@
+"""Functions for migrating PEtab files from different versions"""
+
+import logging
+import pandas as pd
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'pandas'](https://app.codacy.com/manual/dweindl/PEtab/pullRequest?prid=4885513)